src/model/database/db_connection.js

```diff
@@ -25,8 +25,4 @@ const options = {
     max: process.env.MAX_CONNECTIONS || 2,
```

Comment:
`ssl: process.env.hostname !== 'localhost',`
this like doesn't work with me, i found no hostname in the `process.env`
instead you can say
`ssl: params.hostname !== 'localhost',`